apps/Minecraft Pi (Modded)/description

```diff
@@ -1,6 +1,6 @@
 A modded version of Minecraft: Pi Edition with things like survival mode, multiplayer support, and more!
 
-To open the launcher: Menu > Games > gMCPIL or jMCPIL, depending on which version you chose at install time. Use the launcher to set up mods and multiplayer.
+To open the launcher: Menu > Games > Planet Launcher. Use the launcher to set up mods and multiplayer.
```

Comment:
On my system the button is in the "Other" menu category. Could the .desktop file be fixed?

Reply:
I just checked, and those categories are incorrectly specified.
```
[Desktop Entry]
Name=Planet Launcher
Exec=/usr/bin/planet-launcher
Icon=/usr/lib/planet-launcher/assets/logo512.png
Type=Application
Comment=A better launcher
GenericName=Minecraft Pi Launcher
Categories=games, minecraft
Keywords=Minecraft, MCPIL, gMCPIL, jMCPIL, Minecraft Pi, Minecraft launcher, MCPI, MCPI-Revival, mcpiscript 
```
Whoever maintains the Planet Launcher debfile should consider fixing that.